### PR TITLE
[CHORE] Removed unused and duplicated end2end tests and resolve some Gradle dependency conflicts

### PR DESCRIPTION
**Pull request by lijamie98**
### Description

Removed unused and duplicated end2end tests and resolve some Gradle dependency conflicts

### Context

Improve code maintainability.

### Testing

- `./gradlew test`

### CODE REVIEW

File: extended-tests/src/test/kotlin/org/stellar/anchor/platform/test/Sep24BaseEnd2EndTest.kt [hunk 1]
```diff
@@ -267,27 +267,26 @@ class Sep24BaseEnd2EndTest(config: TestConfig, val jwt: String) {
   private suspend fun waitForTxnStatus(
     id: String,
     expectedStatus: TransactionStatus,
+    exitStatus: TransactionStatus,
```
Comment (philipliu) [inline extended-tests/src/test/kotlin/org/stellar/anchor/platform/test/Sep24BaseEnd2EndTest.kt:270]: Maybe make `ERROR` the default value?

**Review by JiahuiWho — APPROVED** [review]

File: build.gradle.kts [hunk 2]
```diff
@@ -144,9 +144,6 @@ subprojects {
     register<Test>("testFireblocksE2E") {
```
Comment (philipliu) [inline build.gradle.kts:144]: Do we still need this anymore?

File: build.gradle.kts [hunk 3]
```diff
@@ -144,9 +144,6 @@ subprojects {
...
       useJUnitPlatform()
 
-      include("**/AnchorPlatformCustodyEnd2EndTest**")
```
Comment (philipliu) [inline build.gradle.kts:147]: Should the `exclude(...)` be removed from the previous `test` configuration?

File: extended-tests/build.gradle.kts [hunk 4]
```diff
@@ -14,7 +14,8 @@ dependencies {
   testImplementation(libs.okhttp3.mockserver)
   testImplementation(libs.assertj.core)
   testImplementation("org.springframework.boot:spring-boot-starter-web")
-
+  testImplementation(libs.kotlin.serialization.json)
```
Comment (philipliu) [inline extended-tests/build.gradle.kts:17]: Why wasn't this required before this change?

Comment (lijamie98) [reply extended-tests/build.gradle.kts:17]: There is a dependency conflict with the Wallet SDK. This is to force the dependency version.

File: build.gradle.kts (hunk 2 above)
Comment (lijamie98) [reply build.gradle.kts:144]: I am not sure but I don't think it is being used. I will clean up.

File: build.gradle.kts (hunk 3 above)
Comment (lijamie98) [reply build.gradle.kts:147]: I have removed the `testFireblocksE2E`

File: extended-tests/src/test/kotlin/org/stellar/anchor/platform/test/Sep24BaseEnd2EndTest.kt (hunk 1 above)
Comment (lijamie98) [reply extended-tests/src/test/kotlin/org/stellar/anchor/platform/test/Sep24BaseEnd2EndTest.kt:270]: Yeah, good idea.

**Review by philipliu — APPROVED** [review]
LGTM

**Pull request merged**